UPDATING.md

```diff
@@ -24,6 +24,34 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Change of method signature in `GCPTransferServiceHook`
+
+The signature of the `create_transfer_job` method in `GCPTransferServiceHook`
+class has changed. The change does not change the behavior of the method.
+Old signature:
+```python
+def create_transfer_job(self, description, schedule, transfer_spec, project_id=None):
+```
+New signature:
+```python
+def create_transfer_job(self, body):
```

Comment:
Może zapewnimy wsparcie dla dwóch sygnatur jednocześnie. Jest to technicznie możliwe. W przyszłości się to dropnie.

Reply:
Myślę, że to za duża przeróbka na ten moment (wymaga dodatkowego otestowania). Zdecydowaliśmy się na podejście z body i tego bym się trzymał.